File: v4/python/audio_stream/main.py
import requests

url = 'http://api-audiostream-sh.fengkongcloud.com/audiostream/v4'
access_key = '{ACCESS_KEY}'
stream_url = '{URL}'
bt_id = '{BT_ID}'
uid = '{UID}'

payload = {
    'accessKey': access_key,
    'appId': 'default',
    'eventId': 'audio',
    'type': 'POLITY_EROTIC_MOAN',
    'btId': bt_id,
    'callback': 'https://jsonplaceholder.typicode.com/posts/',
    'data': {
        'url': stream_url,
        'tokenId': uid,
        'btId': bt_id,
    }
}

res = requests.post(url, json=payload)
# 通过 code 和 riskLevel 判断返回，具体请参考接口文档
if res:
    print(res.json())

File: v4/lua/audio_stream/main.lua
local json = require('cjson')
local hc = require('lib.httpclient').new()

local url = 'http://api-audiostream-sh.fengkongcloud.com/audiostream/v4'
local accessKey = '{ACCESS_KEY}'

local payload = json.encode({
  accessKey = accessKey,
  appId = 'default',
  eventId = 'audio',
  type = 'POLITY_EROTIC_MOAN',
  callback = 'https://jsonplaceholder.typicode.com/posts/',
  data = {
    url = '{URL}',
    tokenId = '{UID}',
    btId = '{BT_ID}',
  }
})

local res = hc:post(url, payload)
local data = json.decode(res['body'])
-- 通过 code 和 riskLevel 判断返回，具体请参考接口文档
print(res['body'])
